test/ctia/http/routes/bundle_test.clj

```diff
@@ -187,7 +187,8 @@
                            indicators
                            sightings)]
     (testing "Import bundle with all entity types"
-      (let [new-bundle (deep-dissoc bundle-maximal :id)
+      (let [new-bundle (-> bundle-maximal
+                           deep-dissoc-entity-ids)
```

Comment:
I think you can remove the thread here and juste use `(deep-dissoc-entity-ids bundle-maximal)`.